cmd/snappy/cmd_build.go

```diff
@@ -56,23 +53,12 @@ func (x *cmdBuild) Execute(args []string) (err error) {
 		args = []string{"."}
 	}
 
-	snapPackage, err := snappy.BuildSquashfsSnap(args[0], x.Output)
-	if err != nil {
-		return err
-	}
-
-	_, err = exec.LookPath(clickReview)
-	if err != nil {
-		fmt.Fprintf(os.Stderr, "Warning: could not review package (%s not available)\n", clickReview)
-	} else {
-		cmd := exec.Command(clickReview, snapPackage)
-		cmd.Stdout = os.Stdout
-		cmd.Stderr = os.Stderr
-		// we ignore the error for now
-		_ = cmd.Run()
+	if _, err := exec.LookPath("snapcraft"); err != nil {
+		fmt.Fprintf(os.Stderr, `please "sudo apt install snapcraft"`)
```

Comment:
Rather than prescribing the solution, which might not work depending on where it's run, maybe something like:

`"error: build command needs snapcraft available"`

?